assets/src/frontend/components/Home.vue

```diff
@@ -304,11 +304,20 @@
                                     </td>
                                     <td class="action"><span class="flaticon-cancel-music" @click.prevent="removeCustomerNote"></span></td>
                                 </tr>
-                                <tr class="pay-now" @click="initPayment()">
```

Comment:
Please check this scenario inside function `initPayment()` and return if conditions is false